tests/integration/helpers/wrap_kakarot.py

```diff
@@ -123,10 +171,14 @@ async def _wrapped(contract, *args, **kwargs):
 # Example: get_contract("Solmate", "ERC721") will load the ERC721.sol file in the tests/integration/solidity_contracts/Solmate folder
 # Example: get_contract("StarkEx", "StarkExchange") will load the StarkExchange.sol file in the tests/integration/solidity_contracts/StarkEx/starkex folder
 #
-def get_contract(contract_app: str, contract_name: str) -> Contract:
+def get_contract(
```

Comment:
I am seeing this warning when running tests locally and I assume it is because I added an optional argument here:
```
/home/jhnn/.cache/pypoetry/virtualenvs/kakarot-mTn7Smmv-py3.9/lib/python3.9/site-packages/marshmallow/fields.py:945: RemovedInMarshmallow4Warning: The 'default' argument to fields is deprecated. Use 'dump_default' instead.
    super().__init__(**kwargs)
```
Not sure how to resolve this. 

This optional argument was so I could use `get_contract` to get the contract object of `ContractRevertsOnMethodCall` which is one of two contracts defined in the file `RevertTestCases`